### PR TITLE
Add WebURL to users.

### PR DESCRIPTION
**Pull request by ruimiguelsantos**
Adds the missing field to User.

https://docs.gitlab.com/ee/api/users.html#single-user

### CODE REVIEW

File: users.go [hunk 1]
```diff
@@ -87,6 +87,7 @@ type User struct {
 	PrivateProfile            bool               `json:"private_profile"`
 	SharedRunnersMinutesLimit int                `json:"shared_runners_minutes_limit"`
 	CustomAttributes          []*CustomAttribute `json:"custom_attributes"`
+	WebURL                    string             `json:"web_url"`
```
Comment (svanharmelen) [inline users.go:90]: Please place this field above `CreateAt` so it follows the ordering in the docs (which I try to use as guideline here). Thanks!

Comment (svanharmelen) [reply users.go:90]: And it seems some tests also need to be updated to make Travis pass...

Comment (ruimiguelsantos) [reply users.go]: Sure, sorry about that. I've moved the field to honour the docs ordering and updated the failing tests.

**Review by svanharmelen — APPROVED** [review]
Thanks @ruimiguelsantos! LGTM!

**Pull request merged**